dashboard/db/migrate/20201117160845_create_user_ml_models.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUserMlModels < ActiveRecord::Migration[5.1]
+  def change
+    create_table :user_ml_models do |t|
```

Comment:
Currently the production database server is configured with `utf8` as the default character set and collation, and because this migration does not explicitly specify the character set, this table will get created with `utf8`, but your local development database server has the default setting `utf8mb4` configured on the `dashboard` database so there's a diff between the `schema.rb` being committed and what will be generated on `production`. The `staging` database was rebuilt recently, also has 'utf8mb4' now as it's default, and is the authoritative creator/committer of `schema.rb`. @wjordan is working on updating the production configuration to eliminate this, but in the short term, we may want to explicitly specify the character set in this migration. Looking up how to do that ...

Reply:
[Here's an example](https://github.com/code-dot-org/code-dot-org/blob/5975463436e6a99fa8b9f051606544a3056f4e20/dashboard/db/migrate/20200518181001_convert_foorm_submissions_to_utf8mb4.rb#L3) of how to change the default character set after the table has been created.  I don't see a way to specify this directly with the rails `create_tasble` method.